src/app/fyle/add-edit-expense/add-edit-expense.page.ts

```
@@ -1363,16 +1363,12 @@
 
             this.source = 'MOBILE';
 
-            if (orgUserSettings.currency_settings && orgUserSettings.currency_settings.enabled) {
-              if (orgUserSettings.currency_settings.preferred_currency) {
-                etxn.tx.currency = orgUserSettings.currency_settings.preferred_currency;
-              }
-            } else if (
+            if (
               orgSettings.org_expense_form_autofills &&
               orgSettings.org_expense_form_autofills.allowed &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)